tutorials/tts/FastPitch_Adapter_Finetuning.ipynb

```diff
@@ -645,17 +647,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0908431a",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "wave_model = WaveformFeaturizer(sample_rate=sample_rate)"
```

Comment:
why was this removed? This object is needed for converting wav -> features

Reply:
We don't need to use `reference_spec` during inference, so we don't need to get the audio features. The only thing we need is the model and text.